labgrid/driver/usbsdwiredriver.py

```diff
@@ -37,3 +39,19 @@ def set_mode(self, mode):
             self.mux.control_serial,
         ]
         processwrapper.check_output(cmd)
+
+    @Driver.check_active
+    @step(title='sdmux_get')
+    def get_mode(self):
+        cmd = self.mux.command_prefix + [
+            self.tool,
+            "-e",
+            self.mux.control_serial,
+            "-u",
+        ]
+        proc = subprocess.run(
```

Comment:
I think you should use the `processwrapper` here.

Reply:
Updated.  I was using subprocess here, since the sdmux is using subprocess in the same place.  I can create a new pull requests for sdmux after this one, but I'll need someone else to test, since I don't have an sdmux.